qutebrowser/mainwindow/mainwindow.py

```diff
@@ -43,7 +43,7 @@
 
 
 def get_window(via_ipc, force_window=False, force_tab=False,
-               force_target=None):
+               force_target=None, force_raise=None):
     """Helper function for app.py to get a window id.
 
     Args:
```

Comment:
Please document `force_raise` in the docstring.

If I'm getting this right, `True` forces raising, `False` forces no raising, and `None` uses `should_raise`? Is that really needed? What about a `no_raise=True` or so, which only suppresses raising?

Reply:
Yes, `no_raise` would be enough here.

Reply:
Added docstring here and in `app.open_url`. The decision to transform it to `no_raise` is up to you, I think.

Reply:
Let's do `no_raise` then, no point in making this more complex without needing it :wink:

Reply:
Will do.